### ISSUE TITLE
Use monodepth2 checkpoint pretrained on kitti ? 

### ISSUE THREAD

**Issue by Yang-Hao-Lin** [madhubabuv/AllDay-DeMoN#1]
Hi 
Great work on all  day monocular depth prediction. 

According to Figure 4 (c) in your paper, the loss at step 1 seems to be smaller than random initialization. Did you start your training default with monodepth2 checkpoint trianed on kitti ?

THX

**Comment by madhubabuv** [comment]
Hello!

No, we did not use monodepth2 pre-trained models. That loss drop was observed because of the denoised images used during the photometric loss calculation.
